core/stores/redis/hook_test.go

```diff
@@ -16,10 +16,11 @@ import (
 
 func TestHookProcessCase1(t *testing.T) {
 	ztrace.StartAgent(ztrace.Config{
-		Name:     "go-zero-test",
-		Endpoint: "http://localhost:14268/api/traces",
-		Batcher:  "jaeger",
-		Sampler:  1.0,
+		Name:        "go-zero-test",
+		Endpoint:    "http://localhost:14268/api/traces",
+		Batcher:     "jaeger",
+		Sampler:     1.0,
+		OtlpHeaders: map[string]string{},
```

Comment:
no need add OtlpHeaders here cause    this field is optional

Reply:
Thks, fixed at  https://github.com/zeromicro/go-zero/pull/3219/commits/ba4769cd250d9510680dd7b18636f68693990968